jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java

```diff
@@ -19,6 +19,9 @@
  * Abstract implementation of a dialect. We recommend that every dialect should extend this abstract class.
  */
 public abstract class AbstractSqlDialect implements SqlDialect {
+    protected static final List<String> SHARED_PROPERTIES = Arrays.asList(SQL_DIALECT_PROPERTY,
```

Comment:
I'm not sure about the name. Maybe COMMON_PROPERTIES or STANDARD_PROPERTIES would be better?

Reply:
Actually that is the remnant of an experiment. I am going to remove that line. With Java 9 there might be a more elegant way to express the relationship between common and dialect-specific properties.

We should review all immutable lists with the Java 9 port anyway.